### PR TITLE
Bulgarian translation from Crowdin

### PR DESCRIPTION
**Pull request by magul**
Beta version as it was not yet proofread.

Translation made by:
* Tsvetelina Koleva (https://crowdin.com/profile/tsvetelinak0)
* Vadim Kolosov (https://crowdin.com/profile/VadKo)
* Vanyo Georgiev (https://crowdin.com/profile/vanyog)

### CODE REVIEW

**Comment by magul** [comment]
@tsvetelinak0 are you working on that? Or should I just merge it and you will review any improvements to the initial version?

**Comment by tsvetelinak0** [comment]
will have a look right now!

**Comment by magul** [comment]
@tsvetelinak0 the idea is to apply improvements in Crowdin and then reimport them here. Otherwise, we would be constantly overwriting whatever we improve here.

**Comment by tsvetelinak0** [comment]
of course that makes perfect sense. will do my suggestions in Crowdin that make more sense re translation

**Comment by tsvetelinak0** [comment]
I deleted all suggested comments in github and made thorough check in Crowdin translation + made some improvements where needed. Hope it is taking better shape now?

**Comment by magul** [comment]
@tsvetelinak0 I've just refreshed this pull request, you can go for another round of review here. Thanks in advance!

File: bg/django_start_project/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+> Част от тази глава е базирана на уроци от Geek Girls Carrots (https://github.com/ggcarrots/django-carrots).
+> 
+> Части от тази глава се основават на [django-marcador tutorial](http://django-marcador.keimlink.de/), лицензиран съгласно лиценза Creative Commons Attribution-ShareAlike 4.0 International. Урокът за django-marcador е защитен с авторско право от Markus Zapke-Gründemann и сие.
+
+Ще създадем малък блог!
+
+Първата стъпка е да започнете нов проект на Django. По принцип това означава, че ще изпълним някои скриптове, предоставени от Django, които ще създадат скелета на проект на Django за нас. Това е само куп директории и файлове, които ще използваме по-късно.
+
+Имената на някои файлове и директории са много важни за Django. Не трябва да преименувате файловете, които предстои да създадем. Преместването им на друго място също не е добра идея. Джанго трябва да поддържа определена структура, за да може да намери важни неща.
```
Comment (tsvetelinak0) [inline bg/django_start_project/README.md:11]:
```suggestion
Имената на някои файлове и директории са много важни за Django. Не трябва да преименувате файловете, които предстои да създадем. Преместването им на друго място също не е добра идея. Django трябва да поддържа определена структура, за да може да намери важни неща.
```

File: bg/django_models/README.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    │   ├── settings.py
+    │   ├── urls.py
+    │   └── wsgi.py
+    ├── myvenv
+    │   └── ...
+    └── requirements.txt
+    
+    
+
+След създаването на приложение също трябва да кажем на Django, че трябва да го използва. Правим това във файла `mysite/settings.py` -- отваряме го във вашия редактор на кодове. Трябва да намерим `INSTALLED_APPS` и да добавим ред, съдържащ `'blog.apps.BlogConfig',` най-отгоре `]`. Така че крайният продукт трябва да изглежда така:
```
Comment (tsvetelinak0) [inline bg/django_models/README.md:101]:
```suggestion
След създаването на приложение също трябва да кажем на Django, че трябва да го използва. Правим това във файла `mysite/settings.py` -- отваряме го във вашия редактор на кодове. Трябва да намерим `INSTALLED_APPS` и да добавим ред, съдържащ `'blog.apps.BlogConfig'`. Така че крайният продукт трябва да изглежда така:
```

**Comment by magul** [comment]
@tsvetelinak0 we still need that to be applied in Crowdin.

**Comment by tsvetelinak0** [comment]
has been done! applied in Crowdin!

**Comment by magul** [comment]
@tsvetelinak0 refreshed, one another round of review or do we merge it in that shape?

**Comment by tsvetelinak0** [comment]
I think it can be merged now. I am not a proofreader and all I have done make sense to me as a coder (also been through this tutorial couple of times)

**Comment by magul** [comment]
Great!, I've just merged it, thank you for your contribution here!

**Comment by tsvetelinak0** [comment]
Fantastic! happy to help!

**Pull request merged**